index.js

```diff
@@ -294,6 +294,38 @@ class Statham {
       });
     });
   }
+
+  /**
+   * Search and return keys and values that match given string.
+   *
+   * @param {String} phrase
+   *
+   * @returns {{}}
+   */
+  search(phrase) {
+    let tmp  = {};
+    let data = this.data;
+
+    if (this.isModeNested()) {
+      data = flatten(this.data);
+    }
+
+    Object.getOwnPropertyNames(data).forEach(key => {
+      if (typeof data[key] === 'string' && data[key].indexOf(phrase) !== -1) {
```

Comment:
Switch conditional. If array, make searchTarget JSON.stringify() and search that.